internal/cri/server/service.go

```diff
@@ -180,6 +185,87 @@ type CRIServiceOptions struct {
 	Client *containerd.Client
 }
 
+func (c *criService) initTracing() error {
+	if c.config.Tracing == nil {
+		log.L.Info("Tracing is still nil after loading from env, tracing is disabled")
+		return nil
+	}
+
+	if !c.config.Tracing.Enabled {
+		log.L.Info("Tracing is disabled via config")
+		return nil
+	}
+
+	log.L.Infof("Initializing tracing with config: Enabled=%v, SamplingRate=%f",
```

Comment:
```suggestion
	log.L.Infof("Initializing tracing with config: SamplingRate=%f",
```

It's already enabled.